pages/profile-edit/profile-edit.jsx

```diff
@@ -169,49 +169,44 @@ class ProfileEdit extends React.Component {
     return (
       <div className="row">
         <div className="col-12">
-          <h1>Your profile</h1>
-          <p>
+          <h2 className="h2-heading">Your profile</h2>
+          <p className="mb-4">
             Tell everyone about yourself, so we can connect and collaborate!
           </p>
           <div className="mb-4">
-            <div className="mb-3">
-              <span>
-                Email <em>(email won't be displayed or shared)</em>
+            <h5 className="h5-heading email mb-1">
+              Email <span className="hint-text">(email won't be displayed or shared)</span>
+            </h5>
+            <div className="d-flex flex-column flex-md-row">
+              <p className="mb-1">{user.email}</p>
+              <span className="d-block d-sm-inline-block ml-0 ml-md-4">
```

Comment:
This is within a flexbox so `block` or `inline-block` doesn't make much difference. To make things slightly more semantic we can change this wrapper from

```html
<span className="d-block d-sm-inline-block ml-0 ml-md-4">...</span>
```

to

```html
<div className="ml-md-4">...</div>
```